Src/WindowsServer/WindowsServer.Shared/FirstChanceExceptionStatisticsTelemetryModule.cs

```diff
@@ -220,13 +220,22 @@ private void TrackStatistics(string type, string operation, string method, bool
 
         private string GetDimCappedString(string value, ConcurrentDictionary<string, int> capValues)
         {
-            if (capValues.Count > 100)
+            int tmp;
+            var result = capValues.TryGetValue(value, out tmp);
+
+            if (result)
+            {
+                return value;
+            }
+            else if (capValues.Count > 100)
```

Comment:
If it's a concurrent code, the capValues.Count > 100 might return true and false for the same value:
- Thread A enters the method with value "ABC", capValues.Count = 100; "ABC" is not in the dictionary
- Thread B enters the method with value "ABC", capValues.Count = 100; "ABC" is not in the dictionary
- Whoever adds it first (say "A") might create a case where both "A" and "B" passed TryGetValue() and it returned nothing. Then "A" added it and capValues.Count is now 101, so "B" does not pass the check and gets "OtherValue" instead. Is this a positive concurrency on purpose or this code is not executed in parallel?